specification/hugr.md

```diff
@@ -123,18 +123,23 @@ carry an edge weight:
   ordering. They have no edge weight.
 - `Value` edges carry typed data at runtime. They have a *port* at each end, associated
   with the source and target nodes. They have an `AnyType`as an edge weight.
-- `Static` edges are similar to `Value` edges but carry static data (knowable at
-  compilation time). They have a `CopyableType` as an edge weight.
+- `Const` edges are similar to `Value` edges but carry static data (knowable at
+  compilation time). These has as edge weight a `CopyableType`.
+- `Function` edges refer to a statically-known function, but with a type scheme
+  that (unlike values) may be polymorphic---see [Polymorphism](#polymorphism).
 - `ControlFlow` edges represent possible flows of control from one part of the
   program to another. They have no edge weight.
 - `Hierarchy` edges express the relationship between container nodes and their
   children. They have no edge weight.
 
-`Value` and `Static` edges are sometimes referred to as *dataflow* edges.
+It is useful to introduce some terms for broader classes of edge:
+* *Static* edges are the union of the `Const` and `Function` edges
+* *dataflow* edges are the union of `Value` and Static (thus, `Value`, `Const` and `Function`)
```

Comment:
```suggestion
* *Static* edges are the union of the `Const` and `Function` edges.
* *Dataflow* edges are the union of `Value` and Static (thus, `Value`, `Const` and `Function`).
```

Reply:
Yes, I guess so. Quite a few existing uses of "dataflow edges" thus also capitalized.

Reply:
Actually I was only suggesting to make each bullet here a full sentence, hence the capitalization. But capitalizing Dataflow more generally seems reasonable.